tractor/_trionics.py

```diff
@@ -281,7 +281,6 @@ async def _open_and_supervise_one_cancels_all_nursery(
                     # Instead try to wait for pdb to be released before
                     # tearing down.
                     if is_root_process():
-                        log.exception(f"we're root with {err}")
```

Comment:
This slipped through in `.alhpa1`; was originally for debugging the root clobbering child-in-tty stuff..